File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    removeDuplicates(nums) {
        let prevNum = 0;

        const newSet = new Set();

        nums.forEach(num => {
            newSet.add(num)
        })
        
        const uniqueArray = [...newSet];
        for (let i = 0; i < uniqueArray.length; i++) {
            nums[i] = uniqueArray[i];
        }
        console.log(nums)
        return uniqueArray.length;
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums)
        
        newArr = [0] * length * 2
       
        for idx, num in enumerate(nums):
            newArr[idx] = num
            newArr[idx + length] = num
            
        return newArr